index.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
       10
     ],
     'consistent-return': 0,
-    'curly': 0,
+    'curly': 2,
```

Comment:
👍 

https://eslint.org/docs/rules/curly

Reply:
Perhaps multi-line as well? I believe this is just error.

Reply:
I think we want curly braces always, and specifying those rules would allow for single line statements without curly braces. I may be wrong though.